### PR TITLE
update compiler to 0.19.1

### PR DESCRIPTION
**Pull request by harrysarson**
Upgrading requires a hack to work around compiler error when importing non-local kernel models.

I am not sure it is worth it.

### CODE REVIEW

**Comment by harrysarson** [comment]
@drathier can you purge the cache please :angel:

**Comment by drathier** [comment]
Done, and restarted

**Comment by harrysarson** [comment]
Test failure is legit, some sort of sub par version of sed on mac.

**Comment by drathier** [comment]
LMK when you want a review and/or merge

**Comment by harrysarson** [comment]
I would love a review! I think this is the _best_ patch out there to upgrade the compiler to 0.19.1. However, tests/make.sh contains such a nasty collection of hacks that it may be better to keep using elm 0.19.0.

File: tests/make.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+# Check that this script is run from top level directory
+if [[ $(git rev-parse --show-toplevel) != $(pwd) ]]; then
+  >&2 echo "Please run ./tests/make.sh from git root directory!"
+  exit 1
+fi
+
+FILE_WITH_IMPORT=src/Test/Internal.elm
+
+sed -i.bak 's/import Elm\.Kernel\.Debug/-- import Elm\.Kernel\.Debug/g' "$FILE_WITH_IMPORT"
```
Comment (drathier) [inline tests/make.sh:13]: Is this workaround for this bug? https://github.com/elm/compiler/issues/2123

Comment (harrysarson) [reply tests/make.sh:13]: Yes, the other work around would be to permanently delete this import. However, I think it is better to leave the import there as it self documents the use of kernel code.

**Comment by drathier** [comment]
Do we expect to find/avoid any bugs/issues by moving to 0.19.1? It seems like a pretty hacky change. I haven't worked in this code base for a while, so I don't remember what we should be thinking about here.

**Comment by harrysarson** [comment]
There is not much difference between elm 0.19.0 and 0.19.1 so unlikely to make much difference. Keeping our dependencies up to date is my main motivation. It also seems like a bit of an issue if the elm test runner's test suite cannot pass with the most recent release of elm.

**Comment by drathier** [comment]
@rtfeldman @mgold any input?

**Comment by rtfeldman** [comment]
I don't have strong feelings about this one way or the other...one the one hand, it's not like we're getting a big benefit, but on the other hand, it's two commits, so easy to roll back if we regret it.

I'd say feel free to merge if you like, @harrysarson!

**Comment by harrysarson** [comment]
@drathier I can squash my commits before we merge this.

**Comment by drathier** [comment]
Aight, imma merge this then :)

**Comment by harrysarson** [comment]
> @drathier can you purge the cache please 👼

Thanks for merging! Please purge the cache for master!

**Comment by drathier** [comment]
Thanks for your help! Appreciate the link too :) Caches are nuked, builds restarted, and master is green!

**Pull request merged**